### ISSUE TITLE
EcoPath Connection

### ISSUE THREAD

**Issue by MKLau** [SEELab/enaR#27]
Shiela,

The easiest way to install the R package is to download and install in from within R.  When you launch R, there will be an option in the menu bar to install packages.  You can select the enaR package.  Alternatively, you can enter the following at the command line:

install.packages("enaR")

R will ask you to select a CRAN mirror -- just choose a site near you.  

The only other snag is that to install the most recent version of enaR, you will need to have at least version 3 of R.  You can see which version of R you have by typing

version

at the command line.  

Please let me know if you have any other problems.

Stuart
# 

=             Stuart Borrett                                                               = 
=  http://people.uncw.edu/borretts                                          = 
= http://uncw.academia.edu/StuartBorrett                             =
# = https://www.researchgate.net/profile/Stuart_Borrett/       =

"A system is never the sum of its parts, its the product of their interactions"  Russell Ackoff (http://bit.ly/gpu6d2)

On Oct 22, 2013, at 11:59 AM, Sheila Heymans Sheila.Heymans@sams.ac.uk wrote:

Hi Stuart, Jeroen

I am a bit rusty –but somehow I cannot figure out how to get the R script from the cran website? I think I got further than this last time but cannot seem to find the R file to run?

If I go to the Windows binary: enaR_2.0.zip file it doesn’t give me the run_enaR R script file that I need? Where do I get that? I am an R virgin you know J

Or should I be going somewhere else on the R enaR website?

Sheila

From: Stuart R. Borrett [mailto:stuart.borrett@gmail.com] 
Sent: 18 October 2013 12:37
To: Sheila Heymans
Cc: Jeroen Steenbeek; v.christensen@fisheries.ubc.ca
Subject: Re: EwE-Environ

Sheila,

Thank you so much for re-starting this conversation.  I would really like to finalize the connection between Ecopath and enaR.  

To update you, we released a major update to enaR this summer that fixed a few important bugs and provides us a stable base for future updates. The R vignette describes the update (http://cran.r-project.org/web/packages/enaR/vignettes/enaR.pdf) -- though I don't expect the changes should not affect the connection to Ecopath and our previous work.    

In addition, we now have a draft manuscript prepared to introduce the software.  Part of why you were on my mind is that this connection is stated as future work in the discussion section.  

I cannot meet today, but please let me know what I need to do to facilitate the next steps.

Also, congratulations on your recent PLoS One paper on the Baltic Sea -- I look forward to reading it.  

Stuart
# 

=             Stuart Borrett                                                               = 
=  http://people.uncw.edu/borretts                                          = 
= http://uncw.academia.edu/StuartBorrett                             =
# = https://www.researchgate.net/profile/Stuart_Borrett/       =

"A system is never the sum of its parts, its the product of their interactions"  Russell Ackoff (http://bit.ly/gpu6d2)

On Oct 18, 2013, at 4:11 AM, Sheila Heymans Sheila.Heymans@sams.ac.uk wrote:

Ok, lets try and chat today … I don’t have any meetings booked, so skype me when you get a chance.

S

From: Jeroen Steenbeek [mailto:drmbongo@gmail.com] 
Sent: 17 October 2013 19:42
To: Sheila Heymans; v.christensen@fisheries.ubc.ca
Cc: Stuart Borrett (stuart.borrett@gmail.com)
Subject: Re: EwE-Environ

Dear all,
This is where the discussion stalled back in February. While we seek clarification on Respiration I'd like to get you up and running with the plug-in, Sheila.
Jeroen

On 02/04/2013 18:14, Sheila Heymans wrote:
Hi Villy, Stuart

I really need to talk to you Villy.

Stuart I am waiting for that before we can continue.

Cheers
Sheila

From: Sheila Heymans 
Sent: 02 April 2013 15:45
To: v.christensen@fisheries.ubc.ca
Cc: drmbongo@gmail.com; Stuart Borrett (stuart.borrett@gmail.com)
Subject: EwE-Environ

Hi Villy, Jeroen

Stuart and I have just been going through the data extraction procedure and have some questions.

Jeroen, Villy -  the SCOR file that we get out gives us the diet matrix and not the flows – we need the consumption matrix out not the diet matrix. That is the easy one.

Stuart, I figured out what I did wrong with the R calculations. The way they calculate R is from the Basic inputs – see file attached … it comes from the basic inputs and basic estimates sheets. And it is fine, but there is some problem with the calculations …

Villy, Jeroen – there is something wrong with the Respiration calculations because somehow you get different values for R than the way we would calculate it. Not sure if it has something to do with the detritus fate – which for this model has some export …

Stuart, skype me when you get this … I can walk you through it.

Villy/Jeroen, I need to figure out how you get the R.

S

Sheila JJ Heymans PhD FSB
SAMS
Scottish Marine Institute
Oban
Scotland
PA371QA
+44 (0)1631 559418
http://www.sams.ac.uk/sheila-heymans/

The Scottish Association for Marine Science (SAMS) is registered in Scotland as a Company Limited by Guarantee (SC009292) and is a registered charity (9206). SAMS has an actively trading wholly owned subsidiary company: SAMS Research Services Ltd a Limited Company (SC224404). All Companies in the group are registered in Scotland and share a registered office at Scottish Marine Institute, Oban Argyll PA37 1QA. The content of this message may contain personal views which are not the views of SAMS unless specifically stated. Please note that all email traffic is monitored for purposes of security and spam filtering. As such individual emails may be examined in more detail.

The Scottish Association for Marine Science (SAMS) is registered in Scotland as a Company Limited by Guarantee (SC009292) and is a registered charity (9206). SAMS has an actively trading wholly owned subsidiary company: SAMS Research Services Ltd a Limited Company (SC224404). All Companies in the group are registered in Scotland and share a registered office at Scottish Marine Institute, Oban Argyll PA37 1QA. The content of this message may contain personal views which are not the views of SAMS unless specifically stated. Please note that all email traffic is monitored for purposes of security and spam filtering. As such individual emails may be examined in more detail.

The Scottish Association for Marine Science (SAMS) is registered in Scotland as a Company Limited by Guarantee (SC009292) and is a registered charity (9206). SAMS has an actively trading wholly owned subsidiary company: SAMS Research Services Ltd a Limited Company (SC224404). All Companies in the group are registered in Scotland and share a registered office at Scottish Marine Institute, Oban Argyll PA37 1QA. The content of this message may contain personal views which are not the views of SAMS unless specifically stated. Please note that all email traffic is monitored for purposes of security and spam filtering. As such individual emails may be examined in more detail.
